opacclient/opacapp/src/main/java/de/geeksfactory/opacclient/frontend/MainPreferenceActivity.java

```diff
@@ -23,4 +24,9 @@ protected void onCreate(Bundle savedInstanceState) {
     protected PreferenceFragment getNewMainPreferenceFragment() {
         return new MainPreferenceFragment();
     }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
```

Comment:
Oh, I added it when I tried to fix a problem where selecting a different warning period in the settings didn't have any effect. But the real problem was I needed to return `true` instead of `false` at `MainPreferenceFragment.java:90`, so it can be removed again. Done in 3b60160.